src/dsp_tools/commands/excel2json/project.py

```diff
@@ -86,7 +87,64 @@ def _get_validate_list_folder(data_model_files: str, folder: list[Path]) -> tupl
     return listfolder, processed_files
 
 
-def _get_validate_onto_folder(data_model_files: str, folder: list[Path]) -> tuple[list[Path], list[str]]:
+def excel2json(
+    data_model_files: str,
+    path_to_output_file: str,
+) -> bool:
+    """
+    Converts a folder containing Excel files into a JSON data model file. The folder must be structured like this:
+
+    ::
```

Comment:
is this double colon there intentionally?

Reply:
This is the same way it appears on main, I didn't write it but I assume it serves a purpose.

Reply:
sorry, missed the fact that this is one of the moved functions, not something you wrote.